src/styles/blocks/card.scss

```diff
@@ -0,0 +1,65 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+  box-sizing: border-box;
+  width: 200px;
+  height: 410px;
```

Comment:
Your card's height is too big

Reply:
That's why the tests don't pass. Please refer to figma mock

Reply:
Fixed that, I also needed to add height to image, otherwise tests wouldn't pass